Ginger/GingerCoreCommon/Actions/Act.cs

```diff
@@ -1975,6 +1979,21 @@ public override void PrepareItemToBeCopied()
             this.IsSharedRepositoryInstance = TargetFrameworkHelper.Helper.IsSharedRepositoryItem(this);
         }
 
+        protected override RepositoryItemBase CopyRIObject(RepositoryItemBase repoItemToCopy, List<GuidMapper> guidMappingList, bool setNewGUID)
```

Comment:
is it required override a method. As we publish this nugget to other applications and its core class. Try to add another method for starting dirty check.